File: Composite Pattern/CS/Person.cs
namespace CompositePattern
{
    public abstract class Person
    {
        protected string Name;
        protected int Age;
        protected string Gender;
        
        public string GetName()
        {
            return this.Name;
        }

        public int GetAge()
        {
            return this.Age;
        }

        protected string GetGender()
        {
            return this.Gender;
        }

        public abstract string GetInfo();
    }
}

File: Composite Pattern/PHP/Person.php
<?php 

abstract class Person 
{
    protected int $age;
    protected string $gender;

    public function getAge()
    {
        return $this->age;
    }

    public function getGender()
    {
        return $this->gender;
    }

    public abstract function getInfo();
}

?>